frontend/src/metabase/dashboard/components/AddSeriesModal/QuestionList.jsx

```diff
@@ -83,17 +83,38 @@ export const QuestionList = React.memo(function QuestionList({
 
   const compatibleQuestions = useMemo(
     () =>
-      filteredQuestions?.filter(question =>
-        isQuestionCompatible(visualization, dashcard, dashcardData, question),
-      ),
+      filteredQuestions?.filter(question => {
+        try {
+          return isQuestionCompatible(
+            visualization,
+            dashcard,
+            dashcardData,
+            question,
+          );
+        } catch (e) {
+          console.warn(
+            `Could not check question compatibility for a question with id=${question?.id?.()}`,
```

Comment:
Should we be adding console messages for things like this?

Reply:
It will help us identify ids of problematic questions, which is a part of https://github.com/metabase/metabase/issues/26507

Reply:
❤️